api/v1/client/src/commonMain/kotlin/OrtServerClient.kt

```diff
@@ -59,7 +59,7 @@ class OrtServerClient(
 
             val client = createOrtHttpClient(JSON) {
                 defaultRequest {
-                    url(config.baseUrl)
+                    url(config.baseUrl.takeIf { it.endsWith("/") } ?: "${config.baseUrl}/")
```

Comment:
Commit message: Isn't this more of a conventional thing than a requirement? Like, you could very well use no trailing slash here, but then you'd need to use a leading slash in e.g. `post` calls. Simply put, strings are just concatenated as-is, and you need to decide where to put the slash.

Reply:
> Commit message: Isn't this more of a conventional thing than a requirement?

No.
Using
```
defaultRequest {
    url(config.baseUrl)
}
```

Without a trailing slash, e.g. with the url `https://example.com/ort`, it will remove the `/ort` part and therefore point to a different URL.

Reply:
See the documentation of `DefaultRequest`:
```
 *     * Base URL doesn't end with slash ->
 *       remove last path segment of base URL and concat strings.
 *       Example:
 *       base = `https://example.com/dir/deafult_file.html`,
 *       request = `file.html`,
 *       result = `https://example.com/dir/file.html`
```